### ISSUE TITLE
optionsGroup: don't suffix group name to label during edit/form

### ISSUE THREAD

**Issue by mstaack** [outl1ne/nova-multiselect-field#216]
isn't this redundant during edit/form view?

code:
```
            Multiselect::make('Demo', 'demo_id')->options([
                1 => [
                    'label' => 'Var 1',
                    'group' => 'Group 1'
                ],
                2 => [
                    'label' => 'Var 2',
                    'group' => 'Group 1'
                ],
                3 => [
                    'label' => 'Var 3',
                    'group' => 'Group 2'
                ],
                4 => [
                    'label' => 'Var 4',
                    'group' => 'Group 2'
                ],
                5 => [
                    'label' => 'Var 4',
                    'group' => 'Group 2'
                ]
            ])->nullable()->singleSelect()->hideFromIndex(),
```

result:
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/10169509/236193986-c5d89769-a323-46eb-b93e-9e3e947d02bd.png">

source:
https://github.com/outl1ne/nova-multiselect-field/blob/main/resources/js/mixins/HandlesFieldValue.js#L82

**Comment by Tarpsvo** [comment]
Heya! Fixed in version 4.3.1. It was indeed unnecessary to display them twice. :)

**Comment by mstaack** [comment]
brilliant!
